Homework #16: Intro to Selenium WebDriver

KhrysCS opened the pull request:
Homework #16: Intro to Selenium WebDriver

I created a test case for registrationNavigation.

Added a new class named Homework16 which contains the test case for registrationNavigation.

Modified TestNG.xml: added Homework16 class

KhrysCS committed:
Update LoginTests.java

Homework #15: Java Git

Changed the URL from https://qa.koel.app/" to https://testpro.io/".

KhrysCS committed:
Homework #16: Intro to Selenium WebDriver

Homework #16: Intro to Selenium WebDriver

Created a test case for registrationNavigation.

Added new class named Homework16 which contains the test case for registrationNavigation.

Modified TestNG.xml: added Homework16 class

KhrysCS committed:
Merge pull request #1 from KhrysCS/homework-16

Homework #16: Intro to Selenium WebDriver

Berichicko commented:
Hello,

I have checked your homework 16, and I wanted to let you know that you're doing great! Keep up the excellent work!
Best wishes from the HW team (Alex, [aliaksandr@testpro.io](mailto:aliaksandr@testpro.io))!

![804deb64dc406a20a75da05d91a1a69e](https://user-images.githubusercontent.com/104761482/235961479-9afb2841-b657-46c9-99f8-2402f933b843.jpg)

![image](https://github.com/Test-Pro-Bootcamp/A51-QA-Automation-Framework/assets/104761482/ff15d029-f6cb-4be3-8b42-7cddd0ba63f6)

KhrysCS committed:
Lesson17 Demos and Homework #17: TestNG Overview

Added ProfileTests.java: Demo class from lesson 17

Added Homework17.java: added method to test adding song to playlist. Completed all functionality. Still need to refactorize.

Modified BaseTest.java: added helper methods
Modified LoginTests.java: refactored class to use helper methods from BaseTest

Modified TestNG.xml: added Homework17 and ProfileTests

KhrysCS committed:
Homework #17: TestNG Overview Completed

Refactorized Homewor17.java and added comments.

KhrysCS committed:
Merge pull request #2 from KhrysCS/homework-17

Homework #17: TestNG Overview

KhrysCS committed:
A51-L18-Class-Demo

Some practice code from lesson 18 and files to keep notes and .

KhrysCS committed:
Homework #18: XML File Configurations

Homework #18: XML File Configurations

Added Test class Homework18 .java the test class contains the playSong test case
Added Test suit  SmokeHW18.xml file for smoke testing the new test class by method.
Modified the build.gradle file to invoke the SmokeHW18.xml file for testing

KhrysCS committed:
Merge pull request #3 from KhrysCS/homework-18

Homework 18

KhrysCS committed:
Update README file

KhrysCS committed:
Homework #19: Additional Annotations & DataProviders

Modified build.gradle to use TestNG.xml
Modifed BaseTest.java to add @Parameter BaseURL and assign parameter to url string.
Modified TestNG config file to add parameter BaseURL and added class name Homework19
Added Homework19.java test class for test case deletePlaylist

KhrysCS committed:
Update ReadMe file

KhrysCS committed:
Merge pull request #4 from KhrysCS/homework-19

Homework 19

KhrysCS committed:
Merge pull request #5 from KhrysCS/homework-15

Update LoginTests.java

KhrysCS committed:
Update LoginTests.java

KhrysCS committed:
Homework #20: Selenium Waits

Homework #20: Selenium Waits

Modified BaseTest.java - Replaced timeouts with explicit waits
Added Homeword20.java - Replaced timeouts with explicit waits
Modified TestNG.xml - Added Homeword20 class

KhrysCS committed:
Create Homework20Draft.java

KhrysCS committed:
Revert "Create Homework20Draft.java"

This reverts commit c7bae0d28194468b545da835d2acc0c4b9a0c32b.

KhrysCS committed:
Merge pull request #6 from KhrysCS/homework-20

Homework 20

KhrysCS committed:
A51-L21-Class-Demo

A51-L21-Class-Demo
Added HomeTest.java which contains exercises from lesson 21

KhrysCS committed:
Merge pull request #7 from KhrysCS/A51-L21-Class-Demo

A51-L21-Class-Demo

KhrysCS committed:
Homework #21: Web Elements +  Action Class

Homework #21: Web Elements + Action Class
Added Homework21.java to create rename playlist test case using actions and waits

KhrysCS committed:
Merge pull request #8 from KhrysCS/homework-21

Homework #21: Web Elements +  Action Class

KhrysCS committed:
A51-L22-Class-Demo

A51-L22-Class-Demo
Added BasePage.java, HomePage.java, LoginPage.java to implement POM design pattern.
Modified BaseTest and LoginTest to use POM classes.

KhrysCS committed:
Merge pull request #9 from KhrysCS/A51-L22-Class-Demo

A51-L22-Class-Demo

KhrysCS committed:
Homework #22: Selenium POM

Homework #22:Selenium POM
Modified HomeTest.java: added a method for test case play song that uses the pom design pattern to login to page.

KhrysCS committed:
Merge pull request #10 from KhrysCS/homework-22

Homework #22: Selenium POM

KhrysCS committed:
Update Homework20.java

Edited Homework20.java class to add a song to the playlist and then delete the song added,

KhrysCS committed:
Merge pull request #11 from KhrysCS/homework-20

Update Homework20.java

KhrysCS committed:
Update Homework20.java

Modified Homework 20 to remove Thread.sleep

KhrysCS committed:
Merge pull request #12 from KhrysCS/homework-20

Update Homework20.java

KhrysCS committed:
A51-L23-Class-Demo

Contains the practice code from the slides from lesson 23.

KhrysCS committed:
Merge pull request #13 from KhrysCS/A51-L23-Class-Demo

A51-L23-Class-Demo

KhrysCS committed:
Homework #23: Selenium Page Factory

Added classes: HomeWork22Page.java and HomeWork22.java to implement Page Factory and use @Finby annotation and Fluent interfaces.
Modified classes:  LoginTests and HomePage to implement Page Factory and use @Finby annotation and Fluent interfaces

KhrysCS committed:
Merge pull request #14 from KhrysCS/homework-23

 Homework #23: Selenium Page Factory

KhrysCS committed:
Homework #24: Selenium Grid

Modified build.gradle, BaseTest.java, TestNG.xml to implement Browser Factory Method

KhrysCS committed:
Merge pull request #15 from KhrysCS/homework-24

Homework #24: Selenium Grid

KhrysCS committed:
Update BaseTest.java

Added comment to BaseTest.java to remember command to run on command prompt to start selenium-grid

KhrysCS committed:
Merge pull request #16 from KhrysCS/A51-L25-Class-Demo

Update BaseTest.java

KhrysCS committed:
Homework #25: Selenium Parallel Testing

Homework #25:
Selenium Parallel Testing

KhrysCS committed:
Merge pull request #17 from KhrysCS/homework-25

Homework #25: Selenium Parallel Testing

KhrysCS committed:
A51-L26-Class-Demo

Creating Branch A51-L26-Class-Demo

KhrysCS committed:
Update build.gradle

KhrysCS committed:
Homework #27: Selenium Cucumber

Homework #27:Selenium Cucumber

KhrysCS committed:
Merge pull request #18 from KhrysCS/homework-27

Homework #27: Selenium Cucumber

KhrysCS committed:
Merge pull request #19 from KhrysCS/A51-L26-Class-Demo

A51 l26 class demo
